etc/notes/CHANGES_5.0.0.md

```diff
@@ -269,10 +268,6 @@ marked internal and are no longer exported.
 - `PipeOptions`
 - `ServerOptions`
 
-### `DeleteOptions.single` Option Removed
-
-TODO - merge in Neal's removal of `collection.remove` and combine notes
```

Comment:
Discussing this with Neal to make sure it's ok to remove

Reply:
From slack, we discussed, I think it's okay to remove, it's an option that was only respected in .remove which is removed, so it's gone along with that API. It was mistakenly available on the TS of deleteOne/deleteMany, so it was permitted but ignored by those operations. we could add a note to the section about the removal of .remove if we want to capture this